ec/acc/deployment_config_test.go

```diff
@@ -79,8 +81,20 @@ func setDefaultTemplate(region, template string) string {
 	case "gcp":
 		return "gcp-" + template
 	default:
+		return buildAwsTemplate(template)
+	}
+}
+
+func buildAwsTemplate(template string) string {
+	legacyTemplates := []string{defaultTemplate, hotWarmTemplate, ccsTemplate,
```

Comment:
shouldn't this be `nonLegacytemplates`?

Reply:
The new templates are the solution templates (`observability`, `security`). So I figured I'd call the other templates (v2) legacy? But yeah, the name can be misleading, I'll change that name :)

Reply:
Ah makes sense. Thanks!